frontend/src/store/actions/Profile/ProfileActions.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable max-len */
```

Comment:
For now, I think this file shouldn't be included in this PR.

Reply:
I will delete this! Thank you!